tests/eas/acceptance.py

```diff
@@ -28,15 +28,15 @@
 from env import TestEnv
 from test import LisaTest, experiment_test
 
-# Read the config file and update the globals
-CONF_FILE = os.path.join(
-    os.path.dirname(
-        os.path.abspath(__file__)),
-    "acceptance.config")
-
-with open(CONF_FILE, "r") as fh:
-    conf_vars = json.load(fh)
-    globals().update(conf_vars)
+WORKLOAD_DURATION_S = 5
```

Comment:
Let's add at least a single line of comment to mark that this are global test configuration parameters...